module/account/src/Application/Controller/Api/LogController.php

```diff
@@ -46,6 +47,8 @@ public function __construct(LogQueryInterface $query, LogGrid $grid)
     /**
      * @Route("/profile/log", methods={"GET"})
      *
+     * @IsGranted("USER_READ")
```

Comment:
This end point is for the user profile the data, should be available for each logged user, check body of the method.